src/modules/previewpane/PreviewPaneUnitTests/Properties/AssemblyInfo.cs

```diff
@@ -2,11 +2,11 @@
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
-[assembly: AssemblyTitle("PreviewPaneUnitTests")]
+[assembly: AssemblyTitle("UnitTests-MarkdownPreviewHandler")]
 [assembly: AssemblyDescription("")]
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
-[assembly: AssemblyProduct("PreviewPaneUnitTests")]
+[assembly: AssemblyProduct("UnitTests-MarkdownPreviewHandler")]

```

Comment:
nit: This naming is inconsistent with the naming of the project/module libraries. I suggest mdpreviewhandlerTests or something along those lines.

Reply:
All the tests under previewPane follow this naming pattern. Is there any best practice article that enforces this naming style?